src/soot/jimple/spark/internal/TypeManager.java

```diff
@@ -138,6 +147,9 @@ final public void makeTypeMask() {
                 if (sc.isInterface()) {
                     makeMaskOfInterface(sc);
                 }
+                if(!visitedTypes.get(t.getNumber()) && !((RefType)t).getSootClass().isPhantom()) {
```

Comment:
If we didn't visit this type already and we're not phantom, then we need to make sure to process the type here.